charts/konk/templates/configmap.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "konk.fullname" . }}-configmap
+  labels:
+    app: {{ include "konk.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
```

Comment:
Use the `include` for this to keep it consistent with the others.

Reply:
Updated the labels.